src/compiler/python_generator.h

```diff
@@ -54,21 +47,21 @@ struct GeneratorConfiguration {
   std::string implementations_package_root;
 };
 
-class PythonGrpcGenerator : public google::protobuf::compiler::CodeGenerator {
+class PythonGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
  public:
   PythonGrpcGenerator(const GeneratorConfiguration& config);
   ~PythonGrpcGenerator();
 
-  bool Generate(const google::protobuf::FileDescriptor* file,
+  bool Generate(const grpc::protobuf::FileDescriptor* file,
                 const std::string& parameter,
```

Comment:
Sorry, I didn't catch that from the first pass, but you'll also want to change std::string to grpc::string. Sorry :-)